src/libs/antares/study/parts/hydro/series.cpp

```diff
@@ -60,22 +59,21 @@ static uint EqualizeTSsize(TimeSeries& TScollection1,
                            unsigned int height1 = HOURS_PER_YEAR,
                            unsigned int height2 = HOURS_PER_YEAR)
 {
-    const auto& ts1Width = TScollection1.timeSeries.width;
-    const auto& ts2Width = TScollection2.timeSeries.width;
+    const auto ts1Width = TScollection1.timeSeries.width;
+    const auto ts2Width = TScollection2.timeSeries.width;
+    const auto maxWidth = std::max(ts1Width, ts2Width);
 
-    PairOfIntegers pairOfTSsizes(ts1Width, ts2Width);
-
-    if (pairOfTSsizes.bothZero())
+    if (ts1Width == 0 && ts2Width == 0)
     {
         TScollection1.reset(1, height1);
         TScollection2.reset(1, height2);
         return 1;
     }
 
-    if (pairOfTSsizes.same())
-        return pairOfTSsizes.sup();
+    if (ts1Width == ts2Width)
+        return maxWidth;
 
-    if (pairOfTSsizes.bothGreaterThanOne())
+    if (ts1Width > 1 && ts2Width > 1)
```

Comment:
`if (ts1Width > 1 && ts2Width > 1)` ==>` if (maxWidth > 1)`

Reply:
This is not equivalent. For example `max(0, 100) > 1` but `(100>1 && 0>1)=(true && false)=false`.

Reply:
Oops, sorry, I meant `minWidth > 1`. We would have to refine **minWidth**.
Up to you.